src/jobservice/job/impl/gc/garbage_collection.go

```diff
@@ -235,28 +236,32 @@ func (gc *GarbageCollector) deleteCandidates(ctx job.Context) error {
 		if err != nil {
 			return err
 		}
+		gc.logger.Info("start to delete untagged artifact.")
 		for _, art := range untagged {
-			gc.logger.Infof("delete the untagged artifact: ProjectID:(%d)-RepositoryName(%s)-MediaType:(%s)-Digest:(%s)",
-				art.ProjectID, art.RepositoryName, art.ManifestMediaType, art.Digest)
 			if err := gc.artCtl.Delete(ctx.SystemContext(), art.ID); err != nil {
 				// the failure ones can be GCed by the next execution
 				gc.logger.Errorf("failed to delete untagged:%d artifact in DB, error, %v", art.ID, err)
 			}
```

Comment:
`continue` needed here, otherwise line 245 will be logged.

Reply:
fixed.